### ISSUE TITLE
Bug: WalletConnect gets stuck on multiple proposal flow

### ISSUE THREAD

**Issue by plor** [odyssy-automaton/pokemol-web#1849]
https://www.loom.com/share/9194fd0bf84d4723a126636d634b153f

The 'Connect Wallet' button gets stuck on Wallet Connect and forces you to connect a wallet through Wallet Connect and then disconnect in order to get the Metamask option back.

Refreshing doesn't fix it either.

When exiting the Wallet Connect modal the 'Connect Wallet' button should offer both Metamask and Wallet Connect options.

**Comment by wackerow** [comment]
Just noticed this as well... switched to WalletConnect to connect with a safe, and then unable to get back to MetaMask.

Closing this and/or hard refreshing came back with WalletConnect as only option again, unable to get back to MetaMask vs WalletConnect options

Clearing the daohaus.club cookie fixed this.

**Comment by plor** [comment]
@brossetti1 let me know if this is clear enough to work off of. I can go through to detail the steps to reproduce if that will help.

**Comment by brossetti1** [comment]
ill take a look at this one next and let you know if i have any questions

**Comment by brossetti1** [comment]
@plor yeah can you take me through reproduction steps? so far im not able to replicate the issue so far - when I select Wallet Connect or try to connect to a safe minion as in the video, it always rereshes the wallet connection options so metamask or wallet connect are available.

any instructions on this one would be helpful, thanks!

**Comment by plor** [comment]
Will do, it is a bit of a tricky one. Lots of back and forth and such.

**Comment by brossetti1** [comment]
@plor I cannot replicate this one - @santteegt wondering if you know whats going on here?

**Comment by santteegt** [comment]
Hey @plor @brossetti1!

I was unable to reproduce the error. Did several tests on Chrome, Firefox & Brave. I also did a Dao2Dao proposal using the Minion "Use WalletConnect" feature using the same & different browsers tabs for each instance and never got stuck on WalletConnect when trying to switch wallet providers
